frontend/src/components/smart/PvPArenaSummary.vue

```diff
@@ -72,11 +72,20 @@
             <span>{{ tierTopRankers[2] && tierTopRankers[2].rank || 'N/A'}}</span>
           </li>
         </ul>
-        <!-- <a href="/" class="rankings">View all rankings</a> -->
+        <ul class="topPlayersList">
+          <li class="header">
+            <span>Current Season</span><span>Restarts In</span>
+          </li>
+          <li>
+            <span>{{ currentRankedSeason }}</span>
+            <vue-countdown :time="secondsBeforeNextSeason * 1000" v-slot="{ days, hours, minutes, seconds }">
```

Comment:
I see a duplication here, with stuff from PvPArenaPreparation, can we make a component out of it? (everything in < ul >)